### PR TITLE
Wr 295 group keycloak endpoints

### PR DESCRIPTION
**Pull request by rosasck**
Pre warning: Will need to add context to some structs

### CODE REVIEW

File: identityClient/identityClient.go [hunk 1]
```diff
@@ -1297,3 +1299,39 @@ func (c *E3dbIdentityClient) FederatedIdentityKeyCheck(ctx context.Context, para
 	err = e3dbClients.MakeSignedServiceCall(ctx, c.requester, request, c.SigningKeys, c.ClientID, nil)
 	return err
 }
+
+// AccessControlPolicy Enables Access Control for an Application Client, returning error (if any).
+func (c *E3dbIdentityClient) AccessControlPolicy(ctx context.Context, params AccessControlPolicyRequest) (*AccessControlPolicyResponse, error) {
```
Comment (galxy25) [inline identityClient/identityClient.go:1304]: `EnableAccessControlPolicy`, structs can be nouns but methods/functions should be more verby

**Review by galxy25 — APPROVED** [review]

File: identityClient/api.go [hunk 2]
```diff
@@ -1190,3 +1190,39 @@ type FederatedIdentityKeyCheckRequest struct {
 	ClientID                        uuid.UUID `json:"client_id"`
 	PasswordDerivedPublicSigningKey string    `json:"password_derived_public_signing_key"`
 }
+
+// AccessControlPolicyRequest wraps the values required to request Access Control Policy for an Application.
```
Comment (efabens) [inline identityClient/api.go:1194]: I think the following is more accurate.
`AccessControlPolicyRequest wraps the values required to enable or disable an Access Control Policy for an application.`

Comment (rosasck) [reply identityClient/api.go]: Will need to update comments for a lot of the structs and methods!

**Comment by efabens** [comment]
Other naming thoughts, but will hold off until finalized

File: keycloakClient/client_applications.go [hunk 3]
```diff
@@ -0,0 +1,324 @@
...
+	path := c.apiURL.String() + "/auth/admin/realms/" + realmName + "/groups/" + groupId + "/role-mappings/clients/" + clientId
+	request, err := createVanillaRequest("DELETE", path, roleMappings)
+	if err != nil {
+		return err
+	}
+	return makeJSONCall(accessToken, request, nil)
+}
+
+// GetToznyInternalGroupPolicy
+func (c *Client) GetToznyInternalGroupPolicy(accessToken string, realmName string, clientID string) ([]InternalGroupPolicy, error) {
```
Comment (rosasck) [inline keycloakClient/client_applications.go:275]: Added these methods!

**Review by galxy25 — APPROVED** [review]

**Review by galxy25 — APPROVED** [review]

**Review by efabens — APPROVED** [review]

**Pull request merged**